python/python.sh

```diff
@@ -42,7 +42,7 @@ install_python() {
     cd /tmp/${PY_INSTALLER_DIR}
     mkdir -p /${PY_HOME}/lib
     ./configure --prefix=${PY_HOME} --enable-shared LDFLAGS="-Wl,-rpath ${PY_HOME}/lib"
-    make -j "$(nproc)"
+    make -j1
```

Comment:
Not sure if the reason of failure is thread-safe issue, build using single thread for now

See this run: https://github.com/openmerlin/dockerfile/actions/runs/12924519416/job/36046258369

Reply:
It's flaky on ubuntu-24.04